packages/eks-controller/templates/deployment.yaml

```diff
@@ -0,0 +1,25 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: eks-config-controller
+  namespace: cattle-system
+spec:
+  selector:
+    matchLabels:
+      ke.cattle.io/controller: eks-config-controller
```

Comment:
If we are grouping the CRD to KontainerEngine already, can we just do value as `eks`/ `aks`...